cdn-uploader.js

```diff
@@ -0,0 +1,45 @@
+const ftp = require("basic-ftp")
+
+const config = {
+host: process.env.AKAMAI_CDN_HOSTNAME,
+user: process.env.AKAMAI_CDN_USERNAME,
+password: process.env.AKAMAI_CDN_PASSWORD,
+cpCode: process.env.AKAMAI_CDN_CPCODE,
+port: 21,
+basePath: "./builds"
+}
+
+function UploadFilesFromPath(sources) {
+	process.chdir(config.basePath);
+	console.log("[+] Connecting to CDN...");
+	for (let i = 0; i <= sources.length-1; i++) { 
```

Comment:
What do you think about using [for...of](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of)?

Reply:
I like it, I will update it. Thanks !